src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -159,6 +160,15 @@ public static class CLIOptions
             JsonKey = "test-runner"
         };
 
+        public static readonly CLIOption<string> LanguageVersionOption = new CLIOption<string>
+        {
+            ArgumentName = "--language-version",
+            ArgumentShortName = "-lv",
+            ArgumentDescription = $"Set the c# version used to compile. | { FormatOptionsString<LanguageVersion>(_defaultOptions.LanguageVersion, (IEnumerable<LanguageVersion>)Enum.GetValues(typeof(LanguageVersion))) }",
```

Comment:
Do we want to allow all versions? I can imagine that version `1` would not work on most projects. If someone passes `1` as their version value stryker will crash during compilation phase. The error messages shown then could be confusing.

Reply:
If users set the language version to one not actually supported by their project then that's on them

Reply:
Fixed by rewriting active mutation helper to csharp1 language features only